tests/test_selected_word_counter.py

```diff
@@ -181,3 +195,80 @@ def test_no_extract():
                 ), f"{filepath} should contain {word} {count} times"
     finally:
         os.remove(result_output + ".xlsx")
+        for item in os.listdir(test_dir):
+            item_path = os.path.join(test_dir, item)
+            if os.path.isdir(item_path):
+                shutil.rmtree(item_path)
+
+
+def test_count_files_nl():
```

Comment:
test_count_files and test_count_files_nl seem very alike in code, although different in content. Maybe it's easy to remove some code duplication?